python/cudf/cudf/tests/test_parquet.py

```diff
@@ -3407,3 +3408,18 @@ def test_parquet_reader_roundtrip_structs_with_arrow_schema():
 
     # Check results
     assert_eq(expected, got)
+
+
+@pytest.mark.parametrize("chunk_read_limit", [0, 240, 1024000000])
+@pytest.mark.parametrize("pass_read_limit", [0, 240, 1024000000])
+def test_parquet_chunked_reader(chunk_read_limit, pass_read_limit):
```

Comment:
Do you mind fleshing out this test to test some of the other parameters (e.g. row_groups, filtering, etc.)

When I get around to implementing parquet in pylibcudf, I'll probably end up stealing your test :)

Reply:
I added the tests.